README.md

```diff
@@ -53,16 +55,6 @@ make test-project
 
 You may want to see the _Dockerfile_ to check what dependencies need to be installed.
 
-Building with conan and cmake:
```

Comment:
Why  this part was removed?  This  explain how to build nettracer  locally with conan nad cmake.  What is nice.

Reply:
As @pijuszczyk mentioned before, it could be because of the old master version. Will be fixed soon.